tedge/src/cli/connect/command.rs

```diff
@@ -331,13 +331,28 @@ fn new_bridge(
         println!("Checking if tedge-mapper is installed.\n");
 
         if which("tedge_mapper").is_err() {
-            println!("Warning: tedge_mapper is not installed. We recommend to install it.\n");
+            println!("Warning: tedge_mapper is not installed.\n");
         } else {
             service_manager
                 .start_and_enable_service(cloud.dependent_mapper_service(), std::io::stdout());
         }
     }
 
+    match cloud {
+        Cloud::C8y => {
```

Comment:
To better follow the pattern already used in that PR, consider to add a `use_agent` field to the `BridgeConfig` struct.

Reply:
fixed